lib/broadside/configuration/deploy_config.rb

```diff
@@ -24,6 +24,7 @@ class DeployConfig < ConfigStruct
         :instance,
         :lines,
         :predeploy_commands,
+        :bootstrap_commands,
```

Comment:
Think of a standard rails app. You need to run a series of rake tasks when standing up a new staging server, for example. Predeploy commands run before each deploy, i.e. `rake db:migrate`. Bootstrap commands run after the task definition has been created but before you've actually started any containers. You'd run commands like `rake db:create`, etc.